llvm/test/CodeGen/AArch64/sve-streaming-mode-fixed-length-int-to-fp.ll

```diff
@@ -2951,7 +2965,10 @@ define half @ucvtf_i16_f16(ptr %0) {
 ; CHECK-LABEL: ucvtf_i16_f16:
 ; CHECK:       // %bb.0:
 ; CHECK-NEXT:    ldrh w8, [x0]
-; CHECK-NEXT:    ucvtf h0, w8
+; CHECK-NEXT:    ptrue p0.s
+; CHECK-NEXT:    fmov s0, w8
```

Comment:
Are you planning to investigate why the 16-bit GPR loads are not converted to FPR loads?

Reply:
I believe this only affects 16-bit integers, and it happens because they're legalized to i32 before they can be converted to an FPR load.

Reply:
So this is just lowered the same as a i32 to f32 conversion.